docs/new-architecture-app-modules-android.md

```diff
@@ -179,6 +184,52 @@ public class MyApplicationTurboModuleManagerDelegate extends ReactPackageTurboMo
 }
 ```
 
+</TabItem>
+
+<TabItem value="kotlin">
+
+```kotlin
+package com.awesomeproject
+
+import com.facebook.jni.HybridData
+import com.facebook.react.ReactPackage
+import com.facebook.react.ReactPackageTurboModuleManagerDelegate
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.soloader.SoLoader
+
+class MyApplicationTurboModuleManagerDelegate
+protected constructor(
+    reactApplicationContext: ReactApplicationContext?,
+    packages: List<ReactPackage?>?
```

Comment:
The nullability of those two parameters should be investigated inside `ReactPackageTurboModuleManagerDelegate`. I believe this can be changed safely to be:

```
    reactApplicationContext: ReactApplicationContext,
    packages: List<ReactPackage>
```

Reply:
Fixedm able to compile  ReactPackageTurboModuleManagerDelegate.kt class